docs/advanced-topics/openshift.asciidoc

```diff
@@ -16,7 +16,7 @@ This page shows how to run ECK on OpenShift.
 * <<{p}-openshift-apm,Deploy an APM Server instance with a route>>
 * <<{p}-openshift-beats,Grant privileged permissions to Beats>>
 
-NOTE: Only Elasticsearch and Kibana are compatible with the `restricted` https://docs.openshift.com/container-platform/4.1/authentication/managing-security-context-constraints.html[Security Context Constraint]. To run the APM Server on OpenShift you must allow the Pod to run with the `anyuid` SCC as described in <<{p}-openshift-apm,Deploy an APM Server instance with a route>>
+NOTE: Only Elasticsearch and Kibana are compatible with the `restricted` https://docs.openshift.com/container-platform/4.1/authentication/managing-security-context-constraints.html[Security Context Constraint]. To run the APM Server on OpenShift you must allow the Pod to run with the `anyuid` SCC as described in <<{p}-openshift-apm,Deploy an APM Server instance with a route>>.
```

Comment:
@alaudazzi  Do you think we can take the opportunity of this PR to address https://github.com/elastic/cloud-on-k8s/issues/3110 ? This statement should not be true anymore since APM 7.9.

Also I'm wondering if we should include the following cases:
* Beats for which a [dedicated section already exists](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-openshift-beats.html)
* Enterprise Search 7.9 [which is known to be incompatible with Openshift](https://github.com/elastic/cloud-on-k8s/issues/3656).

Maybe we should remove "Only Elasticsearch and Kibana are compatible" to be more explicit about the particular cases.

Reply:
Thank you @barkbay for pointing this out. In this case we should also update the section https://www.elastic.co/guide/en/cloud-on-k8s/1.2/k8s-openshift-apm.html, right?
Would it be better to address this in a separare PR?